debian/sid/debian/patches/add-docs-rst-readme

```diff
@@ -0,0 +1,49 @@
+Description: Add a top-level README for RST documentation
+ This package installes only RST documentation and not yet HTML
+ documentation. For that reason, the existing top-level README needed to
+ be modified. Additionally note that the examples/ directory is available
+ there.
+Author: Ben Albrecht <balbrecht@cray.com>
+Forwarded: not-needed
+diff --git a/doc/debian/README.rst b/doc/debian/README.rst
```

Comment:
We ignore doc/README.rst and install this one instead. It's installed by chapel-doc.install. This line:

```
doc/debian/README.rst  usr/share/doc/chapel/1.15
```

I think it's easier to just make a new README than try to hack on the old one.